src/pyedb/dotnet/edb_core/cell/layout_obj.py

```diff
@@ -89,6 +89,9 @@ def id(self):
     def delete(self):
         """Delete this primitive."""
         self._edb_object.Delete()
+        self._pedb.modeler._primitives = []
+        self._pedb.padstacks._instances = []
+        self._pedb.padstacks._definitions = []
```

Comment:
Should we also do that when on updates `_edb_properties` since the value might be changed ? I'm wondering if this could lead to a cached object that does not contain the correct value this time.

Reply:
everything in the object is not cached. So there is no need to refresh when you edit the property